MagneticField/ParametrizedEngine/plugins/alpaka/ParabolicParametrizedMagneticField.h

```diff
@@ -0,0 +1,46 @@
+/**
+ Description: Utility function to calculate the Magnetic Field on the GPU
+*/
+
+#ifndef DataFormats_EgammaReco_plugins_alpaka_MagneticFieldParabolicPortable_h
+#define DataFormats_EgammaReco_plugins_alpaka_MagneticFieldParabolicPortable_h
+
+#include <Eigen/Core>
+#include "MagneticField/ParametrizedEngine/plugins/ParabolicParametrizedMagneticField.h"
```

Comment:
This file doesn't exist. Did you mean
```suggestion
#include "MagneticField/ParametrizedEngine/src/ParabolicParametrizedMagneticField.h"
```
?

Either case, the header (or the necessary parts) should be placed in `MagneticField/ParametrizedEngine/interface`.

Reply:
Fixed.